NGitLab.Mock/Config/GitLabHelpers.cs

```diff
@@ -723,6 +726,55 @@ public static GitLabProject WithGroupPermission(this GitLabProject project, stri
             });
         }
 
+        /// <summary>
+        /// Add milestone in group
+        /// </summary>
+        /// <param name="group">Group.</param>
+        /// <param name="title">Title (required)</param>
+        /// <param name="configure">Configuration method</param>
+        public static GitLabGroup WithMilestone(this GitLabGroup group, string title, Action<GitLabMilestone> configure)
+        {
+            return Configure(group, _ =>
+            {
+                var milestone = new GitLabMilestone
+                {
+                    Title = title ?? throw new ArgumentNullException(nameof(title)),
+                };
+
+                group.Milestones.Add(milestone);
+                configure(milestone);
+            });
+        }
+
+        /// <summary>
+        /// Add milestone in group
+        /// </summary>
+        /// <param name="group">Group.</param>
+        /// <param name="title">Title (required)</param>
+        /// <param name="id">Explicit ID (config increment)</param>
+        /// <param name="description">Description.</param>
+        /// <param name="dueDate">Due date time.</param>
+        /// <param name="startDate">Start date time.</param>
+        /// <param name="createdAt">Creation date time.</param>
+        /// <param name="updatedAt">Update date time.</param>
+        /// <param name="closedAt">Close date time.</param>
+        /// <param name="configure">Configuration method</param>
+        public static GitLabGroup WithMilestone(this GitLabGroup group, string title, int id = default, string? description = null, DateTime? dueDate = null, DateTime? startDate = null, DateTime? createdAt = null, DateTime? updatedAt = null, DateTime? closedAt = null, Action<GitLabMilestone>? configure = null)
```

Comment:
configure is not used?

Reply:
It is copied from the method with same name for project below. So it's a existing bug. Whatever, I will fix both of them.